presto-docs/src/main/sphinx/release/release-0.179-t.rst

```diff
@@ -4,13 +4,140 @@ Release 0.179-t
+* Improve performance for selective filters by merging pages. This feature can be disabled by setting the config property ``experimental.filter-and-project-min-output-page-row-count`` or the session property ``filter_and_project_min_output_page_row_count`` to 0.
 
-Some window functions taking array or map types (e.g. approx_percentile) were not executing before that patch.
+Bug Fixes
+---------
+* Fix query failure for ``CHAR`` functions :func:`trim`, :func:`rtrim`, and
+  :func:`substr` when the return value would have trailing spaces under
+  ``VARCHAR`` semantics.
+* Fix incorrect results when performing comparisons between values of approximate
+  data types (``REAL``, ``DOUBLE``) and columns of certain exact numeric types
+  (``INTEGER``, ``BIGINT``, ``DECIMAL``).
+* Fix failure in ``EXPLAIN`` for tables partitioned on a ``TIMESTAMP`` column
+* Fix execution of several window functions on array and map types. Without this patch, some window functions taking array or map types (e.g. approx_percentile) were failing.
+* Fix incorrect empty results for tables filtered on ``CHAR(x)``, ``DECIMAL``, ``DATE``, or ``TIMESTAMP`` partition columns.
+* Fix incorrect results when ``optimizer.optimize-metadata-queries`` is enabled for queries involving aggregation over ``TopN`` and ``Filter``.
+
+Security Changes
+----------------
+* Add Kerberos principal matcing rules to file-based system access control.
+* Support secure internal communication between Presto nodes via HTTPS.
+* Support secure internal communication between Presto nodes via LDAP.
+* Support secure internal communication between Presto nodes via Kerberos.
+* ``ROLE`` support for the Hive connector, including ``CREATE ROLE``,
+  ``DROP ROLE``, ``GRANT ROLE``, ``REVOKE ROLE``, ``SET ROLE``, ``SHOW CURRENT ROLES``,
+  ``SHOW ROLES`` and ``SHOW ROLE GRANTS`` commands. #todo add links
+
+Hive Changes
+------------
+* Allow partitions without files for bucketed tables (via ``hive.empty-bucketed-partitions.enabled``)
+* Allow multiple files per bucket for bucketed tables (via 
+  ``hive.multi-file-bucketing.enabled``). There must be one or more files per
+  bucket. File names must match the Hive naming convention.
+* Fix potential native memory leak when writing tables using RCFile.
+* Fix query failure when computing statistics on an unpartitioned table in CDH 5.11.
+* Add a configuration option ``hive.create-non-managed-table-enabled`` that can disable creating external Hive tables (default value is ``true``).
+* Support role management for the Hive connector.
+
+TPC-H Changes
+-------------
+* Add column statistics for schemas ``tiny`` and ``sf1``.
+
+CLI Changes
+-----------
+* Fix an issue that would sometimes prevent queries from being cancelled when
+  exiting from the pager.
+
+Additional Documentation
+-------------------------
+
+* Query Optimizer
```

Comment:
I think there were some more @rschlussel, please check what was there in the previous release.

Reply:
Why do we have release notes for added documentation?  We mostly add documentation for all the features that we add.  We also added documentation for new properties, removed documentation for things that changed, etc.  

e..g last release there was documentation about the query optimizer.  This release the documentation is different because we improved the query optimizer and the documentation reflects the new features.

Reply:
We also have a value-add in that we added documentation for the CLI, etc. I'm not sure it's worth mentioning, though we have done so in the past.

Reply:
I think it's not needed